index.html

```diff
@@ -0,0 +1,208 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <a href="#" class="welcome">welcome</a>
+        <nav>
+            <img src="images/mobile-menu.png" alt="Menu bar" height="24px">
+        </nav>
+    </header>
+    <!-- Welcome page -->
+  <section class="welcome-image">
+        <!-- Welcome message -->
+    <div class="welcome-message">
+    <h1>Hey there. I'm Richard</h1>
+    <h2>I'm a software developer</h2>
+    </div>
+        <!-- End of Welcome message -->
+    <h3>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don't hestiate to contact me.
+        </h3>
+    <!-- Social Media -->
+    <ul class="social-media">
+        <li id="twitter">
+            <a href="https://twitter.com/RichardroiDX">
+                <img src="images/twitter.png" alt="twitter">
+            </a>
+        </li>
+        <li id="in">
+            <a href="https://www.linkedin.com/in/richard-oguzie-ibeh-richard/?originalSubdomain=ng">
+                <img src="images/i.png" alt="i"><img src="images/n.png" alt="n">
+            </a>
+        </li>
+        <li id="medium">
+            <a href="#">
+                <img src="images/medium.png" alt="medium">
+            </a>
+        </li>
+        <li id="github">
+            <a href="https://github.com/XRichardroidX">
+                <img src="images/github.png" alt="github">
+            </a>
+        </li>
+        <li id="angellist">
+            <a href="#">
+                <img src="images/angellist.png" alt="angellist">
+            </a>
+        </li>
+    </ul>
+</section>
+    <!-- End of Social Media -->
+<!-- End of welcome page -->
+<section>
+    <div class="all-projects">
+        <div id="down-arrow">
+            <img src="images/down-arrow.png" alt="down-arrow" height="50px">
+        </div>
+              <!-- My Projects -->
+            <h2 class="works">My Recent works</h2>
+            <div class="i"><img src="images/Indicator.png" alt="Indicator" width="65px"></div>
+            <div class="project">
+                <img src="images/template-background.png" alt="template-background" width="70%">
+                <div class="information">
+                    <h4 class="abc">Multi-Post Stories Gain+Glory</h4>
```

Comment:
It's better to use more descriptive class name, so that other readers can refer easily and understand